src/site/content/en/accessible/control-focus-with-tabindex/index.md

```diff
@@ -142,6 +142,113 @@ You can learn more about them in our guide on
 [screen reader basics](/semantics-and-screen-readers).
 {% endAside %}
 
+{% AssessmentCallout 'Use the drop-down below each code sample to check your understanding of tab order.' %}
+{% Tabs 'Samples for knowledge self check' %}
+{% Tab 'sample' %}
+
+This HTML renders a modal dialog:
+```html
+<div role="dialog" aria-labelledby="dialog-header">
+  <button aria-label="Close"></button>
+  <h2 id="dialog-header">
+    Do you want to allow notifications from this website?
+  </h2>
+  <button>No</button>
+  <button>Yes</button>
+</div>
+```
+
+{% AssessmentHint 'What is the tab order for the elements in the sample?' %}
+1. The **Close** button
+1. The **No** button
+1. The **Yes** button
+
+Only the `<button>` elements are included in the tab order
+because they're the only native HTML form elements.
+To insert other elements into the tab order, you would add a `tabindex` attribute.
+{% endAssessmentHint %}
+
+{% endTab %}
+{% Tab 'sample' %}
+
+```html
+<section tabindex="-1">
+  <h2>Cat facts</h2>
+  <ul>
+    <li>A group of cats is called a <a href="https://m-w.com/dictionary/clowder">clowder</a>.</li>
+    <li>Most cats are <a href="https://www.catfacts.org/catnip.html"> unaffected by catnip</a>.</li>
+  </ul>
+</section>
+```
+
+{% AssessmentHint 'Which elements from the sample are included in the tab order?' %}
+Only the `<a>` elements are included in the tab order.
+
+The `<section>` element is not in the tab order
+because it has a negative `tabindex` value.
+(It can, however, be focused using the `focus()` method.)
+{% endAssessmentHint %}
```

Comment:
One thing that could be nice as we iterate is a thumbs-up / thumbs-down next to each answer to get feedback from users on whether it made sense

Reply:
I'd love to have a mechanism for per-item feedback! You usually need some context to understand how a user is interpreting an item differently than you had intended, so maybe a link to a feedback form might be useful?

Sorry to cite Khan again, but they've figured out a lot of this stuff. Here's what they do:
![image](https://user-images.githubusercontent.com/14199217/72091391-ea0e3580-32c4-11ea-9313-a3554b1261b2.png)

And here's the form after you click the "Report a problem" link:
![image](https://user-images.githubusercontent.com/14199217/72091442-07db9a80-32c5-11ea-9001-bc76d6d75049.png)